config/optional/editor.editor.rich_text.yml

```diff
@@ -42,6 +42,7 @@ settings:
           name: Media
           items:
             - Blockquote
+            - tide_callout
```

Comment:
Just a style thing but Camel Case seems to be the norm here?

Reply:
I followed `tide_media` presumed how we named custom plugin https://github.com/dpc-sdp/tide_core/pull/179/files#diff-d5400ced7d74da36c387e69173307efbL67